.github/workflows/smoke_tests.yml

```diff
@@ -23,20 +23,20 @@ jobs:
       - name: Install dependencies
         run: poetry install --no-ansi
 
-      - name: Prepare `golem-cluster-local.yaml`
+      - name: Prepare `golem-cluster.local.yaml`
```

Comment:
`golem-cluster.local.yaml` is supposed to be a file that actually exists locally before `utils.apply_overrides` is called - and which contains local changes for a specific user - by default, that file gets included in the `apply_overrides` operation as the bottom-most override (the last one that gets applied) ... so, probaby the name of the _output_ file should be different, e.g. `golem-cluster.tests.yaml` ... or something ... (mostly, something other that `.local.yaml`)...